docs/userguide/constraints.md

```diff
@@ -79,6 +79,37 @@ ContinuousLinearConstraint(
 A more detailed example can be found
 [here](../../examples/Constraints_Continuous/linear_constraints).
 
+### Interpoint constraints
```

Comment:
please write all headings in telegraph style, most importantly all main word capitalized

I know its inconsistent in the code, but Adrian and I both use this, so lets make it consistent